server/server.go

```diff
@@ -1,57 +1,30 @@
 package server
 
+import (
+	"path/filepath"
+
+	"github.com/mrunalp/ocid/oci"
+)
+
 const (
 	runtimeAPIVersion = "v1alpha1"
 )
 
 // Server implements the RuntimeService and ImageService
 type Server struct {
-	runtime    ociRuntime
-	sandboxDir string
-	sandboxes  []*sandbox
+	runtime   *oci.Runtime
+	sandboxes []*sandbox
 }
 
 // New creates a new Server with options provided
 func New(runtimePath, sandboxDir string) (*Server, error) {
-	// TODO(runcom): runtimePath arg is unused but it might be useful
-	// if we're willing to open the doors to other runtimes in the future.
-	r := &runcRuntime{}
-	return &Server{
-		runtime:    r,
-		sandboxDir: sandboxDir,
-	}, nil
-}
-
-// TODO(runcom): export? this is being done just because runc shows a 3 line version :/
-// but it might actually be a useful abstraction (?)
-type ociRuntime interface {
```

Comment:
ok to remove this - I added it because it was kind of future proof when I added it but since OCI is standardizing the CLI this looks fine to me